nengo/utils/compat.py

```diff
@@ -30,6 +32,13 @@ def ensure_bytes(s):
         assert isinstance(s, bytes)
         return s
 
+    def replace(src, dst):
+        try:
+            os.rename(src, dst)
+        except OSError:  # Will be raised on Windows if dst exists
+            os.unlink(dst)
```

Comment:
It is not identical depending on the OS. [`os.replace`](https://docs.python.org/3/library/os.html#os.replace) is identical and the name implies that an existing file might be overwritten.